lib/capistrano-db-tasks/database.rb

```diff
@@ -15,6 +16,21 @@ def postgresql?
       %w(postgresql pg).include? @config['adapter']
     end
 
+    def mongo?
```

Comment:
@numbata it seems i forgot to push last changes. Empty `mongo?` method and `require pry`. I'm sorry, will do